src/pages/ValidateLoginPage/index.tsx

```diff
@@ -5,23 +5,35 @@ import Navigation from '@libs/Navigation/Navigation';
 import * as Session from '@userActions/Session';
 import CONST from '@src/CONST';
 import ONYXKEYS from '@src/ONYXKEYS';
+import ROUTES from '@src/ROUTES';
 import type {ValidateLoginPageOnyxNativeProps, ValidateLoginPageProps} from './types';
 
 function ValidateLoginPage({
     route: {
-        params: {accountID, validateCode},
+        params: {accountID, validateCode, exitTo},
     },
     session,
 }: ValidateLoginPageProps<ValidateLoginPageOnyxNativeProps>) {
+    const handleNavigation = () => {
```

Comment:
Can't we move this logic to the `handleExitToNavigation` method and just call that method here?

Reply:
Sorry, I think this part is fine.